app/src/main/kotlin/com/wire/android/mapper/MessageContentMapper.kt

```diff
@@ -74,14 +74,14 @@ class MessageContentMapper @Inject constructor(
         members: List<MemberDetails>
     ): UIMessageContent.SystemMessage? {
         val sender = members.findUser(userId = senderUserId)
-        val isAuthorSelfAction = content.members.size == 1 && senderUserId == content.members.first().id
+        val isAuthorSelfAction = content.members.size == 1 && senderUserId == content.members.first()
```

Comment:
while I am reading it, it is abit unclear for me what we are checking here, perhaps some comment would help here

Reply:
the `MemberChange` model before it had a `Member` field which is only a `UserId`  that was unnecessary since all what is needed is the userid and the nodel `Member` now have a field `Role` (Admin or Member)

Reply:
Still a comment would help here I think 🤔